client/components/Layout/Images/BlurhashImage.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <div class="absolute">
-    <div v-if="!error" ref="img" class="absolute">
-      <blurhash-canvas
-        v-if="hash"
-        key="canvas"
-        :hash="hash"
-        :width="width"
-        :height="height"
-        :punch="punch"
-        class="absolute"
-      />
+  <div ref="imageElement">
+    <div v-if="!error" ref="img">
+      <v-fade-transition>
+        <blurhash-canvas
+          v-if="hash"
+          :hash="hash"
+          :width="width"
+          :height="height"
+          :punch="punch"
+          class="absolute"
+        />
+      </v-fade-transition>
```

Comment:
We don't need a v-fade transition here. On fast connections the images load so fast that the transition is longer than the time that it takes to load the connection, so we want to show the blurhash as soon as possible without any fancy thing. The image is another thing though.

With your branch, the transition when switching tracks is very "abrupt" if that makes sense.